src/pages/ManagePricing/ManagePricing.connector.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { render, act } from '@testing-library/react'
+import { MockedProvider } from '@apollo/react-testing'
+import wait from 'waait'
+import HostPricingQuery from 'graphql/HostPricingQuery.gql'
+import connector from './ManagePricing.connector'
+
+const mockHostPricing = {
+  units: 'cpu',
+  pricePerUnit: '12'
+}
+
+const mocks = [
+  {
+    request: {
+      query: HostPricingQuery
+    },
+    result: {
+      data: {
+        hostPricing: mockHostPricing
+      }
+    }
+  }
+]
+
+describe('connector', () => {
+  it('runs the HostPricingQuery', async () => {
+    let hostPricing
+
+    const MockComponent = props => {
+      hostPricing = props.hostPricing
+      return null
```

Comment:
If explicit null isn't needed anywhere, you can just terminate the function

Reply:
actually you can't. You need to return null from a react component.

Reply:
ah, right :)